src/client/components/ranking-bar/index.jsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import './index.scss';
+import {Menu, Dropdown} from 'semantic-ui-react';
+
+
+export class RankingBar extends React.Component {
+
+    handleItemClick = (event, value) => {
+
+        this.setState({activeItem: value.value});
+
+        console.log(this.state.activeItem)
+    };
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            activeItem: price
+        };
+    }
+
+    render() {
+
+        const {activeItem} = this.state;
+
+        return (
+            <div className="sorting-bar">
+                <Menu widths="3">
+
+                    <Menu.Item
+                        icon="usd"
+                        name="Lowest price first"
+
+                        value="price"
+                        active={activeItem === "price"}
```

Comment:
Since you're already using constants you need to replace string with them everywhere.